src/plugins/ovp/services/JwplayerHasMediaSubscriber.js

```diff
@@ -23,7 +24,7 @@ export default class JwplayerMediaSubscriber extends EventSubscriber {
     data.jwplayerMediaId = node.get('jwplayer_media_id');
     data.jwplayerSyncEnabled = node.get('jwplayer_sync_enabled');
     data.jwplayerSyncedAt = node.has('jwplayer_synced_at')
-      ? moment(node.get('jwplayer_synced_at')).format('MMM DD, YYYY hh:mm A')
+      ? convertReadableTime(Microtime.fromString(String(`${node.get('jwplayer_synced_at')}000000`)))
```

Comment:
@efjacobson - I think we just need to give the util a timestamp and it will format the date properly. Please change it to:
```suggestion
      ? 
convertReadableTime(node.get('jwplayer_synced_at'))
```